server/src/server.ts

```diff
@@ -59,9 +59,7 @@ connection.onInitialize(async (params: InitializeParams): Promise<InitializeResu
 
   workspaceFolder = params.workspaceFolders?.[0].uri.replace('file://', '')
 
-  // If the language server is not started by VSCode, extensionPath and pathToBitbakeFolder can be undefined
-  // We provide alternatives here to prevent crashes when these variables are used
-  const extensionPath = params.initializationOptions?.extensionPath ?? path.join(__dirname, '..')
+  const extensionPath = path.join(__dirname, '..')

```

Comment:
Now this is only used by `setDocPathAndParse()`, you can put it in the function and rename the function accordingly.

Reply:
Done.